py-polars/polars/dataframe/frame.py

```diff
@@ -3095,8 +3095,9 @@ def write_parquet(
 
             tbl = pa.table(data)
 
-            # do not remove this
+            # do not remove this import!
             # needed below
+            import pyarrow.parquet  # noqa: F401
```

Comment:
looks like this was accidentally removed in https://github.com/pola-rs/polars/pull/9592/files 😄